test/efm_simple_model.jl

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
at this point this file is not needed, right?

Reply:
true